build/version.props

```diff
@@ -2,7 +2,7 @@
 <Project>
 <!-- MicrosoftIdentityModelVersion  -->
   <PropertyGroup>
-    <MicrosoftIdentityModelCurrentVersion>8.0.1</MicrosoftIdentityModelCurrentVersion>
+    <MicrosoftIdentityModelCurrentVersion>8.0.3</MicrosoftIdentityModelCurrentVersion>
```

Comment:
should be the next version, not the current version, so the previews are correct.